interface/patient_file/ccr_import.php

```diff
@@ -22,39 +22,34 @@
  * @author  Ajil P M <ajilpm@zhservices.com>
  * @link    http://www.open-emr.org
  */
+require_once(dirname(__FILE__) . "../../globals.php");
 
-
-
-require_once(dirname(__FILE__) . "/../globals.php");
+use OpenEMR\Core\Header;
 
 ?>
 <html>
 <head>
+<?php Header::setupHeader(['opener']); ?>
```

Comment:
Same question w.r.t. globals.php and options.inc.php? I'd love to know what their division of labor is, and what to expect from each of them.